### PR TITLE
[13.0][MIG] purchase_stock_picking_return_invoicing

### PR DESCRIPTION
**Pull request by kittiu**
Standard migration, ready for review.

### CODE REVIEW

File: purchase_stock_picking_return_invoicing/models/purchase.py [hunk 1]
```diff
@@ -147,18 +145,29 @@ def _compute_qty_returned(self):
             ["purchase_line_id", "product_uom"],
             lazy=False,
         )
-        p = self._prefetch
```
Comment (kittiu) [inline purchase_stock_picking_return_invoicing/models/purchase.py]: @pedrobaeza 
AFIK, the prefetching is changed. I am not sure what I did here is correct.

File: purchase_stock_picking_return_invoicing/tests/test_purchase_stock_picking_return_invoicing.py [hunk 2]
```diff
@@ -99,44 +118,34 @@ def test_purchase_stock_return_1(self):
...
+        ctx = self.po.action_view_invoice()["context"]
+        active_model = self.env["account.move"].with_context(ctx)
+        view_id = "account.view_move_form"
+        with Form(active_model, view=view_id) as f:
+            f.partner_id = self.partner
+            f.purchase_id = self.po
+        inv_1 = f.save()
         self.check_values(self.po_line, 0, 5, 0, 5, "invoiced")
-        self.assertAlmostEqual(inv_1.amount_untaxed_signed, 50, 2)
+        self.assertAlmostEqual(inv_1.amount_untaxed_signed, -50, 2)
```
Comment (kittiu) [inline purchase_stock_picking_return_invoicing/tests/test_purchase_stock_picking_return_invoicing.py:129]: @pedrobaeza this part, I need to change sign to pass test. As I seen in the code, it seem so. 
V12: https://github.com/odoo/odoo/blob/12.0/addons/account/models/account_invoice.py#L71
V13: https://github.com/odoo/odoo/blob/13.0/addons/account/models/account_move.py#L1023

**Comment by kittiu** [comment]
This module is ready for review. With small discrepancy which I report to maintainer @pedrobaeza of this module.

Thanks!

File: purchase_stock_picking_return_invoicing/models/purchase_order.py [hunk 3]
```diff
@@ -60,38 +60,34 @@ def _compute_invoice(self):
...
             if not create_refund:
                 result["res_id"] = refunds.id or False
         result["context"]["default_origin"] = self.name
         result["context"]["default_reference"] = self.partner_ref
         return result
 
-    @api.multi
     def action_view_invoice(self):
         """Change super action for displaying only normal invoices."""
         result = super(PurchaseOrder, self).action_view_invoice()
```
Comment (CarlosRoca13) [inline purchase_stock_picking_return_invoicing/models/purchase_order.py:93]: This should be like other `super()` on code. It is not mandatory, but I think that to maintain consistency, it is the most correct.
```suggestion
        result = super().action_view_invoice()
```

**Review by CarlosRoca13 — APPROVED** [review]
Just a non-blocking comment. 

Tested on runbot - LGTM :+1:

**Review by pedrobaeza — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@kittiu can you please rebase and fix the pre-commit change?

**Comment by kittiu** [comment]
> @kittiu can you please rebase and fix the pre-commit change?

Fixed.

**Comment by pedrobaeza** [comment]
Thanks for the rebase. I'm afraid you need to pass pre-commit and include the changes.

**Comment by pedrobaeza** [comment]
I'm afraid Travis is still red

**Comment by kittiu** [comment]
> I'm afraid Travis is still red

The error is from module `account_invoice_pricelist_sale`
```
  File "/home/travis/build/OCA/account-invoicing/account_invoice_pricelist_sale/tests/test_module.py", line 39, in test_main
    invoice = order._create_invoices()
  File "/home/travis/OCB-13.0/addons/sale/models/sale.py", line 639, in _create_invoices
    raise UserError(_('There is no invoiceable line. If a product has a Delivered quantities invoicing policy, please make sure that a quantity has been delivered.'))
```

Locally, I installed every module in this repo and test run `account_invoice_pricelist_sale` but still no error.

**Comment by kittiu** [comment]
@pedrobaeza looks like I have to fix this first. Will make another PR to do that first. https://github.com/OCA/account-invoicing/pull/667/commits/60f6371cb7544514fb5cc978419da5ad23c4797a

**Pull request merged**